pkg/operator/controller/controller.go

```diff
@@ -43,8 +44,17 @@ const (
 	DNSControllerFinalizer = "dns.operator.openshift.io/dns-controller"
 
 	controllerName = "dns_controller"
+
+	NamespaceRunLevelLabel           = "openshift.io/run-level"             // "0"
+	NamespaceClusterMonitoringLabel  = "openshift.io/cluster-monitoring"    // "true"
+	NamespacePodSecurityEnforceLabel = "pod-security.kubernetes.io/enforce" // privileged
+	NamespacePodSecurityAuditLabel   = "pod-security.kubernetes.io/audit"   // privileged
+	NamespacePodSecurityWarnLabel    = "pod-security.kubernetes.io/warn"    // privileged
 )
 
+var DNSNamespaceLabels = sets.NewString(NamespaceClusterMonitoringLabel, NamespaceRunLevelLabel,
+	NamespacePodSecurityAuditLabel, NamespacePodSecurityEnforceLabel, NamespacePodSecurityWarnLabel)
```

Comment:
No need to make this variable public.  I suggest reformatting it as follows and adding godoc:
```suggestion
// managedDNSNamespaceLabels is a set of label keys for labels that the
// operator manages for the operand namespace.
var managedDNSNamespaceLabels = sets.NewString(
	NamespaceClusterMonitoringLabel,
	NamespaceRunLevelLabel,
	NamespacePodSecurityAuditLabel,
	NamespacePodSecurityEnforceLabel,
	NamespacePodSecurityWarnLabel,
)
```